### ISSUE TITLE
post in primo piano sul sito

### ISSUE THREAD

**Issue by aborruso** [SiciliaHub/ods16#6]
Scrivere un post sul sito ODS e lasciarlo come pagina in homepage per le prossime settimane

**Comment by nelsonmau** [comment]
fatto qui, no?

**Comment by aborruso** [comment]
@nelsonmau no, intendo dire fissarlo come opzione, e non lasciare i post ordinati per data
